.github/workflows/build-mudlet.yml

```diff
@@ -40,17 +50,23 @@ jobs:
 
     - name: "Set location of vcpkg dependencies"
       env:
-#         vcpkgResponseFile: ${{github.workspace}}/cmakelists.txt/vcpkg_${{matrix.triplet}}.txt
-          vcpkgResponseFile: ${{github.workspace}}/3rdparty/our-vcpkg-dependencies/.vcpkg-osx-dependencies
+        vcpkgResponseFile: ${{github.workspace}}/3rdparty/our-vcpkg-dependencies/vcpkg-${{matrix.triplet}}-dependencies
       run: |
         echo "::set-env name=vcpkgResponseFile::$vcpkgResponseFile"
       shell: bash
 
-    - name: Install homebrew dependencies
+    - name: (macOS) Install homebrew dependencies
+      if: runner.os == 'macOS'
       run: |
         # these aren't available or don't work well in vcpkg
         brew install pkg-config libzzip libzip
```

Comment:
It is a side issue but `libzzip` is no longer needed on **MacOs** - and if we change the luarock we bundle for the other OS from `luazip` to `lua-zip` it won't be for them either since #3648 went in!